features/step_definitions/brief_response_steps.rb

```diff
@@ -35,6 +35,10 @@
   @brief_response = create_brief_response(@brief['lotSlug'], @brief_id, @supplier['id'])
 end
 
+Given 'that supplier submits their application' do
```

Comment:
This step definition is too vaguely worded and needs clarification - "submits their application" seems like, if anything, it would be to do with applying for a framework, not applying for an opportunity.  Would be better as something like "that supplier submits their response to that brief".

The functional tests are shared by all teams so we need to be really clear what things are doing and what they apply to.  "their application" in the context of briefs team is very different from "their application" in the context of submissions.